src/lib/pelias.js

```diff
@@ -4,16 +4,17 @@ import { Agent } from 'http'
 const { pelias } = global.__vandelay_util_config
 const agent = new Agent({ keepAlive: true })
 
-const makeRequest = async (opts) =>
-  request.get(opts.host)
+const makeRequest = async (opts) => {
+  return await request.get(opts.host)
```

Comment:
`return await` will fail the linter (at least on the product repo, may not have updated this config)